TESTS/configs/mbedtls.json

```diff
@@ -0,0 +1,7 @@
+{
+    "macros": [
+        "MBEDTLS_SELF_TEST",
+        "MBEDTLS_TIMING_C",
+        "MBEDTLS_TIMING_ALT"
```

Comment:
_ALT should be enabled only for HW crypto targets ?

Reply:
Mbed TLS has a number of _ALT macros, but `MBEDTLS_TIMING_ALT` is only to enable timing based on Mbed OS API (i.e. LowPowerTicker/Ticker). This one is not related to crypto capabilities.